src/kademlia/routing_table.cpp

```diff
@@ -177,10 +177,8 @@ std::tuple<int, int, int> routing_table::size() const
 	for (auto const& i : m_buckets)
 	{
 		nodes += int(i.live_nodes.size());
-		for (auto const& k : i.live_nodes)
-		{
-			if (k.confirmed()) ++confirmed;
-		}
+		confirmed += std::count_if(i.live_nodes.begin(), i.live_nodes.end()
```

Comment:
c++ syntax is not helping here, I hope that with `ranges` it could be improved.

Reply:
yeah, there were a few places where using an algorithm would be too much of an obfuscation that I chose not to convert it. I think this is reasonable though